doc/protocol.xml

```diff
@@ -1564,6 +1564,31 @@ OK
       <title>The music database</title>
 
       <variablelist>
+        <varlistentry id="album_art">
+          <term>
+            <cmdsynopsis>
+              <command>albumart</command>
+              <arg choice="req"><replaceable>URI</replaceable></arg>
+              <arg choice="req"><replaceable>OFFSET</replaceable></arg>
+            </cmdsynopsis>
+          </term>
+          <listitem>
+            <para>
+              Searches the directory the file <varname>URI</varname>
+              resides in and attempts to return a chunk of an album
+              art image file at offset <varname>OFFSET</varname>.
+            </para>
+            <para>
+              Uses the filename "cover" with any of ".png, .jpg,
+              .tiff, .bmp".
+            </para>
+            <para>
+              Returns the file size and actual number
+              of bytes read at the requested offset before
+              transmitting the chunk as raw bytes.
```

Comment:
Fair point, I thought this would be inferred based on the [overview](https://www.musicpd.org/doc/protocol/syntax.html#idm5) but I'm happy to make it more explicit.